core/app/models/spree/order_update_attributes.rb

```diff
@@ -0,0 +1,36 @@
+module Spree
+  class OrderUpdateAttributes
```

Comment:
Extracting this into its own class like this seems like a lot of boilerplate for the benefit of extracting just about 10 lines of code, no? Or are we intending to add more stuff to this class later?